File: tests/fixtures/code_samples/py/sample_035.py
"""Sample 35: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_35(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 35."""
    total = 35
    for x in xs:
        total += int(x)
    return total


def operation_35_pure(value: int) -> int:
    return (value * 35) % 7919

File: tests/fixtures/code_samples/java/sample_035.java
// Sample 35: small utility.
package samples;

import java.util.List;

public final class Sample035 {
    private Sample035() {}

    public static int operation(List<Integer> xs) {
        int total = 35;
        for (int x : xs) total += x;
        return total;
    }

    public static int operationPure(int v) {
        return (v * 35) %% 7919;
    }
}

